github/orgs_teams.go

```diff
@@ -114,10 +114,34 @@ func (s *OrganizationsService) GetTeam(ctx context.Context, team int) (*Team, *R
 	return t, resp, nil
 }
 
+// NewTeam represents a team to be created or modified.
+type NewTeam struct {
+	ID           *int      `json:"id,omitempty"`
+	Name         *string   `json:"name,omitempty"`
+	Description  *string   `json:"description,omitempty"`
```

Comment:
Are any of these fields required rather than optional?

If so, since this struct is used for providing input only, we can use values for mandatory fields.

Reply:
Yes, the `ID` field is required. What you are suggesting makes sense.

Reply:
Great.

If you want to see some examples, see [`RequiredStatusChecks`](https://godoc.org/github.com/google/go-github/github#RequiredStatusChecks) and grep for "Required" and "Optional" in comments.